modules/golang/golang

```diff
@@ -1,5 +1,11 @@
 compdef _golang go
 
+function _deps(){
+	app="go"
+
+	$(hash $app 2>/dev/null) || $(myzsh error "Couldn't find application ${app}" && return 1)
```

Comment:
This is really similar to https://github.com/myzsh/myzsh/pull/77. Maybe there needs to be some sort of deps checking in the core?